pip/qsharp/interop/qiskit/jobs/qsjobset.py

```diff
@@ -17,6 +18,7 @@
 
 from .qsjob import QsSimJob, RunInputCallable
 from ..execution import DetaultExecutor
+from ... import telemetry_events
```

Comment:
Weird if it's working. Maybe I just don't understand the syntax well enough. Seems like one dot is current (jobs) dir, two would be 'qiskit' dir, and three would only get you to 'interop' dir. Not sure why it doesn't need four to get to the root from here.

Reply:
Two or more leading dots indicate a relative import to the parent(s) of the current package, one level per dot after the first. The parent of the current package is `qiskit`.  Two levels up (due to three dots) gets us to the root package.

Reply:
Pretty sure 3 dots isn't getting you to the root level. It's getting you to qsharp/interop. However, in qsharp/interop/__init__.py you have `from .. import telemetry_events` so it's finding that re-export.

Reply:
You are correct